test/commands/add.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+require('../setup')
+
+import {setup, itShouldParse} from './share';
+
+contract('add command', function() {
+
+  setup()
+
+  itShouldParse('should call add script with an alias and a filename', 'add', ['node', 'zos', 'add', 'ImplV1:Impl', '--skip-compile'], function(fake) {
```

Comment:
WDYT about accepting a string with the command line input, instead of the already-split argv, just for ease of writeability in the tests? So, instead of `['node', 'zos', 'add', 'ImplV1:Impl', '--skip-compile']`, you could just write `node zos add ImplV1:impl --skip-compile`. This would only require a `split(' ')` in the `itShouldParse` function.

Reply:
i also prepended `node` so the input is exactly as usage `zos add ImplV1:impl --skip-compile`